### PR TITLE
Map sections for Ritzau ingest [BOR-28]

### PR DESCRIPTION
**Pull request by devketanpro**

### CODE REVIEW

File: server/borsen/io/feed_parsers/ritzau.py [hunk 1]
```diff
@@ -23,12 +23,37 @@ class RitzauFeedParser(BaseRitzauFeedParser):
...
+        data = item_xml.xpath("section/text()", namespaces=namespaces)
+        if data:
+            values = self.section_category_filter(data)
+            for val in values:
+                item["anpa_category"].append(val)
+
+        # Add Default Category:
+        item["anpa_category"].append(
+            {
+                "name": "Generelt",
```
Comment (petrjasek) [inline server/borsen/io/feed_parsers/ritzau.py]: would rather read this data from vocabulary, using the `qcode` to select the value

File: server/borsen/io/feed_parsers/ritzau.py [hunk 2]
```diff
@@ -41,5 +66,14 @@ def _category_filter(self, category):
                 populated_categories.append(match)
         return populated_categories
 
+    def sectionID_category_filter(self, category):
+        return self._category_filter(category, "categories", "ritzau_section_id")
```
Comment (petrjasek) [inline server/borsen/io/feed_parsers/ritzau.py]: why is this still mapped? it seems like this should only go to sections now

Comment (devketanpro) [reply server/borsen/io/feed_parsers/ritzau.py]: I discussed with @IvanJelicSF  that we need both mapping, `<section>` to `Sections CV` and `<sectionID>` to `categories CV`

Comment (IvanJelicSF) [reply server/borsen/io/feed_parsers/ritzau.py]: Hm, not really @devketanpro , sorry if I wasn't precise. We map the sectionID to the Sections CV, the same way we did for the Categories CV. There is no mapping to the Categories CV anymore, just adding this one category there. THanks

**Review by petrjasek — COMMENTED** [review]
don't change the package-lock in this PR, that issue is clearly unrelated so we can ignore it

**Review by petrjasek — APPROVED** [review]

**Pull request merged**